pkg/resourcemonitor/resourcemonitor.go

```diff
@@ -445,3 +475,20 @@ func cpuCapacity(topo *ghw.TopologyInfo, nodeID int) int64 {
 	}
 	return int64(logicalCoresPerNUMA)
 }
+
+func addNodeInformerEvent(handler cache.ResourceEventHandlerFuncs) error {
+	c, err := k8shelpers.GetK8sClient("")
+	if err != nil {
+		return err
+	}
+	// node-status-update-frequency
+	factory := informers.NewSharedInformerFactory(c, 0)
+	nodeInformer := factory.Core().V1().Nodes().Informer()
+	nodeInformer.AddEventHandler(handler)
+	ctx := context.Background()
+	factory.Start(ctx.Done())
```

Comment:
do we want to use all the informer/factory object as throwaways or do we want/need to store references to them in the main ResourceMonitor object?

Reply:
Besides adding the event, I didn't see a reason to keep a reference for it.
We can if we think we would have a future use for it. but atm I don't see one

Reply:
I didn't look hard/deep enough, I'm mostly wondering about side effects here. Do we _need_ to keep reference for the code to work? AFAIR no, asking because I didn't look hard/deep enough yet :)

Reply:
I had exactly the same question but I tested it with and without keeping the reference and it worked.
I guess the reason is that it's a shared informer and when we start it, it starts a go routine to keep track of the events that keep the object alive.
At least this is how I understood.

Reply:
ok, good enough for now then!